fe/fe-core/src/main/java/com/starrocks/lake/DeleteLakeTableTask.java

```diff
@@ -60,23 +59,13 @@ public void run() {
         }
         TNetworkAddress address = new TNetworkAddress(backend.getHost(), backend.getBrpcPort());
         try {
-            LakeServiceAsync lakeServiceAsync = BrpcProxy.getLakeService(address);
-            lakeServiceAsync.dropTable(request, new RpcCallback<DropTableResponse>() {
-                @Override
-                public void success(DropTableResponse dropTableResponse) {
-                    try {
-                        LOG.info("Cleared lake table storage, deleting shards");
-                        GlobalStateMgr.getCurrentStarOSAgent().deleteShards(tabletIds);
-                    } catch (DdlException ex) {
-                        LOG.warn("Fail to delete shards: {}", ex.getMessage());
-                    }
-                }
+            LakeService lakeService = BrpcProxy.getLakeService(address);
+            Future<DropTableResponse> future = lakeService.dropTable(request);
+            DropTableResponse response = future.get();
+            if (response != null) {
+                GlobalStateMgr.getCurrentStarOSAgent().deleteShards(tabletIds);
+            }
```

Comment:
```suggestion
            GlobalStateMgr.getCurrentStarOSAgent().deleteShards(tabletIds);
```
`response` would be `null` on success.